content/docs/5.1/reference/architecture/overview.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Overview
+description: Core concepts and architecture behind TiKV.
+menu:
+    "5.1":
+        parent: Architecture
+        weight: 1
+---
+
+This page discusses the core concepts and architecture behind TiKV, including:
+
+* The [APIs](#apis) that applications can use to interact with TiKV
+* The basic [system architecture](#system) underlying TiKV
+* The anatomy of each [instance](#instance) in a TiKV installation
+* TiKV's terminology including [Placement Driver](#placement-driver), [Store](#store), [Region](#region), and [Node](#node)
+* TiKV's [transaction model](#transactions)
+* The role of the [Raft consensus algorithm](#raft) in TiKV
+* The [origins](#origins) of TiKV
+
+## APIs
+
+TiKV provides two APIs that you can use to interact with it:
+
+| API           | Description                                                                           | Atomicity     | Use when...                                                                   |
+|:------------- |:------------------------------------------------------------------------------------- |:------------- |:----------------------------------------------------------------------------- |
+| Raw           | A lower-level key-value API for interacting directly with individual key-value pairs. | Single key    | Your application requires low latency and doesn't use multi-key transactions. |
+| Transactional | A higher-level key-value API that provides snapshot isolation transaction.            | Multiple keys | Your application requires distributed transactions.                           |
+
+## System architecture
+
+The overall architecture of TiKV is illustrated below:
+
+{{< figure
+    src="/img/basic-architecture.png"
+    caption="The architecture of TiKV"
+    alt="TiKV architecture diagram"
+    width="70" >}}
+
+## TiKV instance
+
+The architecture of each TiKV instance is illustrated below:
+
+{{< figure
+    src="/img/tikv-instance.png"
+    caption="TiKV instance architecture"
+    alt="TiKV instance architecture diagram"
+    width="60" >}}
+
+
+## Placement driver (PD)
+
+The TiKV placement driver is the cluster manager of TiKV, which periodically and automatically balances workload and data storage across nodes by moving Regions. This process is called **auto-sharding**.
+
+## Store
+
+A **Store** refers to the storage node in the TiKV cluster (an instance of tikv-server). Each store has a corresponding TiKV instance.
```

Comment:
This is ambiguous to the `Node` section: Is node a tikv instance or the machine that runs the tikv?

Reply:
what about 
```
A **Store** is an instance of tikv-server.
```

Reply:
Modified a little bit:

A **Store** is an instance of a TiKV server in which multiple peers are stored.